From f5c74708951a44bbee611c888899abd3f1c18609 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 2 May 2014 14:37:45 -0400
Subject: [misc] mei: use cl_dbg where appropriate

Message-id: <1399041470-21330-73-git-send-email-prarit@redhat.com>
Patchwork-id: 79362
O-Subject: [RHEL7.1 PATCH BZ 1088953 72/77] mei: use cl_dbg where appropriate
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit 46922186978848d3e555330d462ce89b4e0dfcfc
Author: Tomas Winkler <tomas.winkler@intel.com>
Date:   Sun Mar 16 14:35:55 2014 +0200

    mei: use cl_dbg where appropriate

    use cl_dbg for debug log messages in cl context

    this also removes checkpatch warning:
    quoted string split across lines

    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/hbm.c b/drivers/misc/mei/hbm.c
index e3e9b53..b963b17 100644
--- a/drivers/misc/mei/hbm.c
+++ b/drivers/misc/mei/hbm.c
@@ -329,8 +329,7 @@ int mei_hbm_cl_flow_control_req(struct mei_device *dev, struct mei_cl *cl)
  mei_hbm_hdr(mei_hdr, len);
  mei_hbm_cl_hdr(cl, MEI_FLOW_CONTROL_CMD, dev->wr_msg.data, len);
 
- dev_dbg(&dev->pdev->dev, "sending flow control host client = %d, ME client = %d\n",
-  cl->host_client_id, cl->me_client_id);
+ cl_dbg(dev, cl, "sending flow control\n");
 
  return mei_write_message(dev, mei_hdr, dev->wr_msg.data);
 }
diff --git a/drivers/misc/mei/main.c b/drivers/misc/mei/main.c
index 4b98d31..69ad2d7 100644
--- a/drivers/misc/mei/main.c
+++ b/drivers/misc/mei/main.c
@@ -125,17 +125,11 @@ static int mei_release(struct inode *inode, struct file *file)
  }
  if (cl->state == MEI_FILE_CONNECTED) {
   cl->state = MEI_FILE_DISCONNECTING;
-  dev_dbg(&dev->pdev->dev,
-   "disconnecting client host client = %d, "
-      "ME client = %d\n",
-      cl->host_client_id,
-      cl->me_client_id);
+  cl_dbg(dev, cl, "disconnecting\n");
   rets = mei_cl_disconnect(cl);
  }
  mei_cl_flush_queues(cl);
- dev_dbg(&dev->pdev->dev, "remove client host client = %d, ME client = %d\n",
-     cl->host_client_id,
-     cl->me_client_id);
+ cl_dbg(dev, cl, "removing\n");
 
  mei_cl_unlink(cl);
 
-- 
1.7.1